.github/workflows/schedule.yml

```diff
@@ -147,9 +147,9 @@ jobs:
       - name: Pull CVAT server image
         run: |
           docker pull ${{ steps.meta-server.outputs.tags }}
+          docker tag ${{ steps.meta-server.outputs.tags }} cvat/server:dev
+          #TODO fix tag in Dockerfile.ci
```

Comment:
Could you elaborate? What do you want to fix?

Reply:
tag `local` is only used in this workflow and it seems to only complicate the payplan. Previously prepared images were labeled with `latest` tag and it already led to using wrong images during testing. I only suggest to remove the use of local tag, I think there is no need in it. But I don't want to do it now and suggest to defere it until workflow refactoring in the future, which should solve many problems such as duplicates and others.

Reply:
FWIW, I don't think using `local` actually needs to complicate the pipeline. We have some code to retag images, but I don't think it's necessary; see #8562.